src/grpc.rs

```diff
@@ -75,3 +86,158 @@ impl ClientPolicies for Server {
         })))
     }
 }
+
+fn to_client_policy(svc: &OutboundService) -> Result<proto::ClientPolicy, tonic::Status> {
+    let http_routes = svc
+        .http_routes
+        .iter()
+        .map(|(route_ref, route)| to_http_route(&route_ref, route))
+        .collect::<Result<_, _>>()?;
+
+    Ok(proto::ClientPolicy {
+        fully_qualified_name: svc.fqdn.as_ref().map(|s| s.to_string()).unwrap_or_default(),
+        endpoint: None,
+        protocol: Some(ProxyProtocol {
+            kind: Some(Kind::Detect(Detect {
+                timeout: Default::default(),
+                http_routes,
+            })),
+        }),
+        filters: to_filters(&svc.client_policies)?,
```

Comment:
this needs to convert only the filters defined by ClientPolicies with a ClientPolicyBinding that selects the client pod's labels. see my comment on `to_filters` for details.